Answer:1 refactored using content projection - updated code for closed PR on same challenge.

koundinyab opened the pull request:
Sorry for late response.

Please see the updated code for comments from 
https://github.com/tomalaforge/angular-challenges/pull/245

tomalaforge commented:
Why did you close the previous PR on the same challenge ? This makes it hard to follow. Just continue on the same PR. 

And you need to rebase your branch. See [this doc](https://angular-challenges.vercel.app/guides/rebase/)

koundinyab commented:
> Why did you close the previous PR on the same challenge ? This makes it hard to follow. Just continue on the same PR.
> 
> And you need to rebase your branch. See [this doc](https://angular-challenges.vercel.app/guides/rebase/)

Hi, Sorry for the confusion. I think because the PR was not updated for few weeks(my bad), you have closed it. I could not re-open that and so, I raised a new one. 
Let me create a new branch and raise a new PR. I think if there are any issues that need to be addressed reviewers can find them anyways. This branch has become old and rebase is throwing too many conflicts.
